lib/project_types/node/commands/create.rb

```diff
@@ -57,7 +59,7 @@ def self.help
         {{command:#{ShopifyCli::TOOL_NAME} create node}}: Creates an embedded nodejs app.
           Usage: {{command:#{ShopifyCli::TOOL_NAME} create node}}
           Options:
-            {{command:--title=TITLE}} App project title. Any string.
+            {{command:--name=NAME}} App project name. Any string.
             {{command:--app_url=APPURL}} App project URL. Must be valid URL.
             {{command:--organization_id=ID}} App project Org ID. Must be existing org ID.
```

Comment:
I think we should remove the word "project" from these help entries (and the matching ones for Rails apps), since these concepts map directly to the fields in the dashboard where we say "App name" and "App URL". It's an app project, but the thing you're naming and the URL you're setting is for the app itself, which shows up publicly.

Reply:
☝️ I agree with this!

Reply:
Sounds good, how's this? I also removed it from the org id line:

<img width="605" alt="Screen Shot 2020-04-23 at 2 59 23 PM" src="https://user-images.githubusercontent.com/30087610/80138714-3aa75700-8573-11ea-943c-4cc3278a7b21.png">